site/docs/v1/tech/core-concepts/users.adoc

```diff
@@ -79,4 +81,4 @@ image::user-search-json-query.png[User Search by JSON Query,width=1200,role=shad
 
 ==== Reindex
 
-It is possible for the search index to become out of sync with the database.  In the FusionAuth admin UI, you may navigate to [breadcrumb]#System -> Reindex# to initiate a reindex of all users. This navigation item will only be displayed when the search engine type is configured to `elasticsearch`.
\ No newline at end of file
+It is possible for the search index to become out of sync with the database.  In the FusionAuth admin UI, you may navigate to [breadcrumb]#System -> Reindex# to initiate a reindex of all users. This navigation item will only be displayed when the search engine type is configured to `elasticsearch`.
```

Comment:
I think this sentence was already here, but I would like to re-word it. It sort of sounds like it may be a normal occurrence for the index to be come out of sync with the underlying relational db. 

This feature is really only there for exceptional cases, such as restoring a db to a new system where the search index does not yet exist, or Elasticsearch has failed, or temporarily failed and the the index is missing users. 

In general, even if a temporary outage occurs with Elasticsearch, once we touch the user again, the index will be back in-sync. 

Perhaps we should re-phrase this to indicate if you are instructed by FusionAuth support, or if you stand up FusionAuth with a db dump / restore - you may need to run this operation. 

Once you are in prod, with 500+ thousand users, at some point this operation can be quite taxing on the system. So it should not be run unless necessary.